src/planner/binder/query_node/plan_subquery.cpp

```diff
@@ -22,7 +22,7 @@ static unique_ptr<Expression> PlanUncorrelatedSubquery(Binder &binder, BoundSubq
 	case SubqueryType::EXISTS: {
 		// uncorrelated EXISTS
 		// we only care about existence, hence we push a LIMIT 1 operator
-		auto limit = make_unique<LogicalLimit>(1, 0, nullptr, nullptr);
+		auto limit = make_unique<LogicalLimit>(false, 1, 0, nullptr, nullptr);
```

Comment:
Also create a `<LogicalLimitPercentage>` and leave the regular Limit as-is. That way the distinction between percentage and non-percentage based limits is more obvious and not just a boolean flag.